manifest.py

```diff
@@ -259,6 +259,14 @@ def create_manifest(sources, tileset, license, account, product, date, notes,
     (
         'abcd-efgh',
         err_msg
+    ),
+    (
+        'ABCD.efgh',
+        err_msg
+    ),
+    (
+        'ab_cd.efgh',
+        err_msg
```

Comment:
@pratikyadav @normanb these additions are good, but let's stop here and if we want more, let's use https://hypothesis.readthedocs.io/en/latest/ to generate tileset names for testing. The reasons: 1) hypothesis makes it easy to test more of the space of possible tileset names, and 2) unlike us hypothesis has no blind spots or biases and will really put our code through the wringer :laughing: